modules/apps/site-initializer/site-initializer-liferay-learn/src/main/resources/site-initializer/style-books/learn/frontend-tokens-values.json

```diff
@@ -12,6 +12,11 @@
 		"name": "brandColor1",
 		"value": "#0b5fff"
 	},
+	"btnOutlineHoverColor": {
+		"cssVariableMapping": "btn-outline-hover-color",
```

Comment:
This token does not exist in dialect when I searched. There are a lot of different hover tokens in dialect (48 tokens) did you not find any there you could use?